src/styles/blocks/cut.txt

```diff
@@ -0,0 +1,115 @@
+.contacts {
+  color: $color-white;
+
+  &__test {
+    @include on-tablet {
+      @include page-grid;
+
+      &__wrapper {
+        grid-column: span 3;
+      }
+
+      &__list {
+        grid-column: span 3;
+      }
+    }
+
+    @include on-desktop {
+      @include page-grid;
+
+      &__wrapper {
+        grid-column: span 5;
+      }
+
+      &__list {
+        grid-column: 8/ 13;
+      }
+    }
+  }
+
+  .contact__input {
+    padding-bottom: 16px;
+
+    &--text {
+      grid-column: span 2;
+      height: 48px;
+      border-radius: 40px;
+      border: none;
+      background-color: $color-retangulo-form;
+      line-height: 20px;
+      margin: 1;
+      padding-left: 16px;
+
+      @include on-tablet {
+        grid-column: span 3;
+      }
+
+      @include on-desktop {
+        grid-column: span 4;
+      }
+    }
+  }
+  .contact__input__area {
+    @include page-grid;
+    padding-bottom: 32px;
+
+    &--text-area {
+      grid-column: span 2;
+      height: 120px;
+      border-radius: 16px;
+      border: none;
+      background-color: $color-retangulo-form;
+      font-weight: 400;
+      font-size: 14px;
+      line-height: 20px;
+      padding: 14px 16px;
+
+      @include on-tablet {
+        height: 148px;
+        grid-column: span 3;
+      }
+
+      @include on-desktop {
+        height: 148px;
+        grid-column: span 4;
+      }
+    }
+  }
+
+  &__list {
+    &--normal {
+      margin: 1;
+      font-weight: 400;
+      font-style: 14px;
+      line-height: 19.6px;
+      padding-bottom: 8px;
+    }
+
+    &--especial {
+      margin: 1;
+      font-weight: 500;
+      font-style: 16px;
+      line-height: 22.4px;
+      padding-bottom: 24px;
+
+      &--final {
+        margin: 1;
+        font-weight: 500;
+        font-style: 16px;
+        line-height: 22.4px;
+      }
+    }
+  }
+}
+
+
+<input
+                class="contacts__form--input"
+                type="email"
+                placeholder="Email"
+              />
+              <textarea
+                class="contacts__form--text-area"
+                name="message"
+                placeholder="Message"
+              ></textarea>
```

Comment:
what is this file?

Reply:
I didn't understand the question.
I added the textarea to receive texts. Should it be done like this?